src/setproperties.md

```diff
@@ -40,18 +44,29 @@ julia> setproperties(o, a="A", c="cc")
 S("A", 2, "cc")
 ```
 
-# Overloading
+# Implementation
```

Comment:
https://docs.julialang.org/en/latest/manual/documentation/ mentions `# Implementation` so I changed it. Not that it's super important.

Reply:
The markdown diffs are great! I personally I find "overloading" a bit clearer, but let's be consistent.